source/extensions/quic_listeners/quiche/platform/BUILD

```diff
@@ -76,14 +76,13 @@ envoy_cc_library(
         "quic_reference_counted_impl.h",
         "quic_ptr_util_impl.h",
         "quic_server_stats_impl.h",
-        "quic_string_impl.h",
         "quic_string_piece_impl.h",
+        "quic_test_impl.h",
```

Comment:
quic_test_impl.h depends on "common/common/assert.h", I think it should stay in the envoy_select_quiche part.

Reply:
Moved quic_test.h into envoy_select_quiche then